vip-portal/src/main/resources/vm/wrapper.vm

```diff
@@ -187,7 +203,12 @@ TMP_FOLDER=/tmp/$TMP_FOLDER
 TMP_FOLDER=../$TMP_FOLDER
 #end
 mkdir -p $TMP_FOLDER
+
+#if($imagepath == "")
```

Comment:
does it work if `imagepath` is null ?

Reply:
If $imagepath is null or undefined in, the #if condition will handle it as if it were an empty string (""). 
In VTL, a null value and an empty string are treated similarly in conditions. So I think the statement handles both conditions appropriately.